docs/authentication/social-connections/twitter.mdx

```diff
@@ -43,4 +43,4 @@ Also, enter the **Authorized Redirect URI** that you grabbed in the previous ste
 
 ![Fill in the Callback URI](/docs/images/authentication-providers/twitter/89fc23c0ac843ba74831c669e17c44a31896a8d7-1196x954.png)
 
-Don't forget to click **Apply** in the Clerk dashboard. Congratulations! Social connection with Twitter is now configured for your instance.
+Finally, click **Add connection** so that the settings are applied. Congratulations! Social connection with Twitter is now configured for your instance.
```

Comment:
```suggestion
Finally, select **Add connection** so that the settings are applied. Congratulations! Social connection with Twitter is now configured for your instance.
```